src/js/controllers/preferencesGlobal.js

```diff
@@ -10,12 +10,16 @@ angular.module('copayApp.controllers').controller('preferencesGlobalController',
       this.selectedAlternative = {
         name: config.wallet.settings.alternativeName,
         isoCode: config.wallet.settings.alternativeIsoCode
-      }; 
+      };
       $scope.spendUnconfirmed = config.wallet.spendUnconfirmed;
       $scope.glideraEnabled = config.glidera.enabled;
       $scope.glideraTestnet = config.glidera.testnet;
+      $scope.notifications = config.notifications ? config.notifications.enabled : true;
     };
 
+    if (isCordova) $scope.mobile = true;
```

Comment:
Just a comment there. I think that we should enable the switch only if `isCordova` and (`isMobile.Android()` or `isMobile.iOS`). Not for WP.  

Also take a look at: https://github.com/bitpay/copay/blob/master/src/js/controllers/walletHome.js#L23

These lines have the right way to define the variable `mobile` (using `this`)